packages/snackbar/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@material/react-snackbar",
+  "version": "0.8.3",
+  "description": "Material Components React Snackbar",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "keywords": [
+    "mdc web react",
+    "material components react",
+    "material design",
+    "material snackbar",
+    "materialsnackbar"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/material-components/material-components-web-react.git"
+  },
+  "dependencies": {
+    "@material/button": "^0.41.0",
```

Comment:
remove @material/button and @material/react-ripple. This isn't used in the component